core/src/main/java/apoc/export/json/ImportJson.java

```diff
@@ -46,6 +47,9 @@ public class ImportJson {
     @Context
     public TerminationGuard terminationGuard;
 
+    @Context
+    public URLAccessChecker urlAccessChecker;
```

Comment:
I assume this breaks backwards compatibility, but that's ok perhaps.

Reply:
Yeah, it does. But that's okay, right - we require a matching APOC version with each Neo4j release? How would we usually go about merging something that breaks like this? Core first, then APOC?